MH4U Dex/Constants.h

```diff
@@ -0,0 +1,115 @@
+FOUNDATION_EXPORT NSString *const QuestNameKey;
+FOUNDATION_EXPORT NSString *const QuestCaravanHallKey;
+FOUNDATION_EXPORT NSString *const QuestDangerKey;
+FOUNDATION_EXPORT NSString *const QuestFeeKey;
+FOUNDATION_EXPORT NSString *const QuestHRPKey;
+FOUNDATION_EXPORT NSString *const QuestMapKeyNameKey;
+FOUNDATION_EXPORT NSString *const QuestKeyIndicatorKey;
+FOUNDATION_EXPORT NSString *const QuestObjectiveKey;
+FOUNDATION_EXPORT NSString *const QuestRewardKey;
+FOUNDATION_EXPORT NSString *const QuestStarsKey;
+FOUNDATION_EXPORT NSString *const QuestSubQuestObjectiveKey;
+FOUNDATION_EXPORT NSString *const QuestFirstMonsterKey;
+FOUNDATION_EXPORT NSString *const QuestSecondMonsterKey;
+FOUNDATION_EXPORT NSString *const QuestThirdMonsterKey;
+FOUNDATION_EXPORT NSString *const QuestFourthMonsterKey;
+FOUNDATION_EXPORT NSString *const QuestTypeKey;
+FOUNDATION_EXPORT NSString *const QuestUrgentKey;
+FOUNDATION_EXPORT NSString *const QuestCaravanKey;
+FOUNDATION_EXPORT NSString *const QuestNoneString;
+
+#pragma mark - Quest Drop Constants
+
+FOUNDATION_EXPORT NSString *const QuestDropsFileName;
+FOUNDATION_EXPORT NSString *const QuestDropIDKey;
+FOUNDATION_EXPORT NSString *const QuestDropQuestIDKey;
+FOUNDATION_EXPORT NSString *const QuestDropRowKey;
+FOUNDATION_EXPORT NSString *const QuestDropItemNameKey;
+FOUNDATION_EXPORT NSString *const QuestDropQuantityKey;
+FOUNDATION_EXPORT NSString *const QuestDropPercentKey;
+
+#pragma mark - Monster Drop Constants
+
+FOUNDATION_EXPORT NSString *const MonsterDropsFileName;
+FOUNDATION_EXPORT NSString *const MonsterDropIDKey;
+FOUNDATION_EXPORT NSString *const MonsterDropMethodKey;
+FOUNDATION_EXPORT NSString *const MonsterDropQuantityKey;
+FOUNDATION_EXPORT NSString *const MonsterDropRankKey;
+FOUNDATION_EXPORT NSString *const MonsterDropPercentKey;
+FOUNDATION_EXPORT NSString *const MonsterDropMonsterNameKey;
+FOUNDATION_EXPORT NSString *const MonsterDropMonsterIDKey;
+FOUNDATION_EXPORT NSString *const MonsterDropItemNameKey;
+
+#pragma mark - Area Drop Constants
+
+FOUNDATION_EXPORT NSString *const AreaDropAreaNameKey;
+FOUNDATION_EXPORT NSString *const AreaDropRankKey;
+FOUNDATION_EXPORT NSString *const AreaDropIDKey;
+FOUNDATION_EXPORT NSString *const AreaDropMethodKey;
+FOUNDATION_EXPORT NSString *const AreaDropPercentKey;
+FOUNDATION_EXPORT NSString *const AreaDropQuantityKey;
+FOUNDATION_EXPORT NSString *const AreaDropItemNameKey;
+
+#pragma mark - Damage Zone Cell Constants
+
+FOUNDATION_EXPORT NSString *const BodyPartLabelText;
+FOUNDATION_EXPORT NSString *const CutLabelText;
+FOUNDATION_EXPORT NSString *const ImpactLabelText;
+FOUNDATION_EXPORT NSString *const ShotLabelText;
+FOUNDATION_EXPORT NSString *const KOLabelText;
+FOUNDATION_EXPORT NSString *const FireLabelText;
+FOUNDATION_EXPORT NSString *const IceLabelText;
+FOUNDATION_EXPORT NSString *const DragonLabelText;
+FOUNDATION_EXPORT NSString *const WaterLabelText;
+FOUNDATION_EXPORT NSString *const ThunderLabelText;
```

Comment:
["Always use `NSLocalizedString` for user-facing strings"](https://github.com/vokal/Coding-Standards/tree/master/iOS#internationalization)